Weapons-Guide.md

```diff
@@ -9,17 +9,17 @@ In most games all players are equipped with a pistol and two of their favorite l
-| <img src="images/weapons/smg.png" width="128px"/> | **Submachine Gun** | While hard to control, the *submachine gun* deals very heavy damage at short to medium range. At medium and longer ranges, the SMG can still do damage in bursts, but it is very difficult to maintain accuracy at long ranges at full auto. The SMG has an airbust secondary fire that hurls shrapnel in a moderate radius at a medium distance. While a very useful medium range weapon, the submachine gun is prone to emptying its ammunition quickly if the user is careless, and this can be exploited by attentive opponents. |
-| <img src="images/weapons/flamer.png" width="128px"/> | **Flamer** | Another ammunition-limited weapon, the *flamethrower* can deal immense damage very quickly via its stream of fire which lances out in front of it. The flamethrower is not a weapon for the careless though; the operator can quickly find themselves in the midst of their own weapon's flames if they are not prudent about avoiding them. The flamer also has a secondary attack, capable of launching the player and their opponents into the air; this is especially useful as a sword deterrent. |
-| <img src="images/weapons/plasma.png" width="128px"/> | **Plasma** | The *plasma inductor* has a medium range primary fire with a large hit radius but a slow projectile speed. While it requires good shot-leading skills, its large hit radius means it is capable of consistent, if unexceptional, damage rates. The secondary fire requires charging, but releases a large mass of plasma which clogs corridors and is useful to get a pursuing enemy off one's tail. |
-| <img src="images/weapons/zapper.png" width="128px"/> | **Zapper** | A fairly standard automatic medium-to-long range weapon, the zapper can deal its damage over any range and at a reasonable pace, but is easily overwhelmed at short range by more specialized weapons. The weapon can be zoomed for better accuracy at long range with its secondary fire. |
-| <img src="images/weapons/rifle.png" width="128px"/> | **Rifle** | The longest ranged, highest damaging weapon in the game, the fully charged rifle secondary can kill an opponent in a single headshot at any range, thanks to its scope. However, it is heavy, has limited ammunition, and a rather weak primary fire that cannot hold its own at short range very well. |
+| <img src="images/weapons/sword.png" width="128px"/> | **Sword** | The *energy sword* deals a decent amount of damage with its primary attack and is capable of one shot kills for those capable of timing and aiming its bleed-inducing secondary attack, or, if not fully charge, inducing bleed. The generous hitbox afforded to the primary of the sword makes it a much more tenable melee option than other options. |
```

Comment:
`charged` with a `d`, probably also better to say `a bleed effect` or `a bleed residual`.